cloudfoundry/repo/repo_manager.go

```diff
@@ -138,8 +126,8 @@ func (rm *Manager) GetGithubRelease(ghOwner, ghRepoName, archiveName string, tok
 		return nil, err
 	}
 
-	path := rm.workspace + "/github_releases/" + ghOwner + "/" + ghRepoName
-	if err = os.MkdirAll(path, os.ModePerm); err != nil {
+	path, err := ioutil.TempDir("", "terraform-provider-cf")
```

Comment:
I would rename to terraform-provider-cf to terraform-provider-cloudfoundry as we are planning to rename the provider.

Reply:
i did the renaming